index.js

```diff
@@ -64,7 +71,7 @@ function forwarded(headers, whitelist) {
     if (!(proxies[i].ip in headers)) continue;
 
     ports = (headers[proxies[i].port] || '').split(',');
-    ips = (headers[proxies[i].ip] || '').split(',');
+    ips = (headers[proxies[i].ip] || '').replace(ipListSpaceFilter, ',').split(',');
```

Comment:
I wonder if it's better to use `trim()` here:
```js
ips = (headers[proxies[i].ip] || '').split(',').map(function (ip) { return ip.trim(); });
```
so we can also remove the `trim()` from the whitelist check below.